TBAnnotationClustering-Swift/ViewController.swift

```diff
@@ -43,7 +43,7 @@ class ViewController: UIViewController, MKMapViewDelegate {
         let toRemove = NSMutableSet(set: before)
         toRemove.minusSet(after as Set<NSObject>)
         
-        dispatch_async(dispatch_get_main_queue()) { () -> Void in
+        NSOperationQueue().addOperationWithBlock() {
```

Comment:
Wouldn't the potential problem you're describing cause the app to crash if the code was actually changing the view from another thread? am I'm missing something?